Source/WebCore/css/CSSGroupingRule.cpp

```diff
@@ -165,7 +165,7 @@ RefPtr<StyleRuleWithNesting> CSSGroupingRule::prepareChildStyleRuleForNesting(St
     CSSStyleSheet::RuleMutationScope scope(this);
     auto& rules = m_groupRule->m_childRules;
     for (size_t i = 0 ; i < rules.size() ; i++) {
-        auto& rule = rules[i];
+        Ref rule = rules[i];
```

Comment:
This is only used for comparison on the next line it seems? I think we should just get rid of this local and use `rules[i].get()` on the next line if that's the case.

Reply:
Does the range-for loop work here?